less/slider.less

```diff
@@ -178,7 +178,9 @@
           -moz-box-shadow: none;
           -webkit-box-shadow: none;
           box-shadow: none;
-          outline: 3px solid orange;
+          .accessibility & {
+              outline: 3px solid orange;
```

Comment:
even tough `@focus-outline` is defined in the core, I decided not to use it in the Plugin. Since it might cause issues with Version of the Framework where `@focus-outline` is not yet defined.
The theme does use `@focus-outline` though.